# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1737905349 -3600
# Parent  e3aee64c335fd9d119199f310055a7196b6e6d12
Misc: Allow language pack download from Betterbird site.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -911,9 +911,9 @@ pref("intl.regional_prefs.use_os_locales
 //  *.liveReload - Switching a langpack will change the language without a restart.
 //  *.liveReloadBidirectional - Allows switching when moving between LTR and RTL
 //      languages without a full restart.
 pref("intl.multilingual.enabled", true); // Allow manual or distro installation of langpacks.
-pref("intl.multilingual.downloadEnabled", false); // Don't allow download from Thunderbird site.
+pref("intl.multilingual.downloadEnabled", true);
 pref("intl.multilingual.liveReload", false);
 pref("intl.multilingual.liveReloadBidirectional", false);
 
 // if true, use full page zoom instead of text zoom
diff --git a/mail/components/preferences/messengerLanguages.js b/mail/components/preferences/messengerLanguages.js
--- a/mail/components/preferences/messengerLanguages.js
+++ b/mail/components/preferences/messengerLanguages.js
@@ -435,9 +435,9 @@ var gMessengerLanguagesDialog = {
       onSelect: item => this.availableLanguageSelected(item),
       onChange: item => {
         this.hideError();
         if (item.value == "search") {
-          this.loadLocalesFromAMO();
+          window.top.openContentTab("https://www.betterbird.eu/downloads", "tab");
         }
       },
     });
 
@@ -447,9 +447,10 @@ var gMessengerLanguagesDialog = {
 
     // If the user opened this from the "Search for more languages" option,
     // search AMO for available locales.
     if (search) {
-      return this.loadLocalesFromAMO();
+      window.top.openContentTab("https://www.betterbird.eu/downloads", "tab");
+      return undefined;
     }
 
     return undefined;
   },
